app/src/pages/playground/PlaygroundInput.tsx

```diff
@@ -1,11 +1,36 @@
 import React from "react";
 
-import { Card } from "@arizeai/components";
+import { Card, Flex, TextArea } from "@arizeai/components";
+
+import { usePlaygroundContext } from "@phoenix/contexts/PlaygroundContext";
+import {
+  selectDerivedInputVariables,
+  selectInputVariableKeys,
+} from "@phoenix/store";
 
 export function PlaygroundInput() {
+  const variables = usePlaygroundContext(selectDerivedInputVariables);
+  const variableKeys = usePlaygroundContext(selectInputVariableKeys);
+  const setVariableValue = usePlaygroundContext(
+    (state) => state.setVariableValue
+  );
   return (
     <Card title="Input" collapsible variant="compact">
```

Comment:
I would just remove the card here, the accordion header already shows that it's input